apps/web/components/dashboard/preview/BookmarkPreview.tsx

```diff
@@ -68,6 +70,27 @@ export default function BookmarkPreview({
   initialData?: ZBookmark;
 }) {
   const { t } = useTranslation();
+  const [activeTab, setActiveTab] = useState<string>("content");
+  const [showTabBar, setShowTabBar] = useState<boolean>(true);
```

Comment:
```suggestion
  const [showTabBar, setShowTabBar] = useState(true);
```
`useState` can infer that its boolean from `true`
same applies for the other similar cases as well